bin/packages/watch.js

```diff
@@ -117,8 +120,9 @@ function isWatchableFile( filename, skip ) {
 function getBuildFile( srcFile ) {
 	// Could just use string.replace, but the user might have the project
 	// checked out and nested under another src folder.
-	const packageDir = srcFile.substr( 0, srcFile.lastIndexOf( '/src/' ) );
-	const filePath = srcFile.substr( srcFile.lastIndexOf( '/src/' ) + 5 );
+	const srcDir = `${ path.sep }src${ path.sep }`;
+	const packageDir = srcFile.substr( 0, srcFile.lastIndexOf( srcDir ) );
+	const filePath = srcFile.substr( srcFile.lastIndexOf( srcDir ) + 5 );
 	return path.resolve( packageDir, 'build', filePath );
```

Comment:
On Windows, `srcFile` parameter includes  backslashes.
The path cannot be sliced correctly because forward slashes are used inthe argument of `lastIndexOf`.
Here I used `path.sep` to change it to cross-platform delimiter.